action.yml

```diff
@@ -1,22 +1,36 @@
+# Reference: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions
+---
+author: Simon Li
 name: major-minor-tag-calculator
 description: Calculate major and minor semver tags, e.g. for tagging containers
-author: Simon Li
+
+# Manually keep the input-, and output- descriptions in sync with the readme!
 inputs:
   githubToken:
-    required: false
-    description: GitHub token
+    required: true
```

Comment:
I changed this to false in my last PR since this will be missing in PRs, and we've got decent handling now.... do you mind keep this as `false` and correcting the README instead?

Reply:
Absolutely, but I'm not confident on the situation to write the update yet. Perhaps you could push a commit clarifying it?

Reply:
I've just pushed a couple of commits, are you ok with them?

Reply:
Ah yes, now I understand how it can be optional! Nice!